src/components/content/resource/helpers/history-entry.tsx

```diff
@@ -84,9 +91,7 @@ const DetailSummary = ({
       aria-label="details"
       onClick={() => {
         setIsDetailShown(!isDetailShown);
-        trackInteraction("expand_detail", {
-          direction: isDetailShown ? "collapse" : "expand",
-        });
+        trackInteraction(isDetailShown ? "collapse_history_entry" : "expand_history_entry");
```

Comment:
This is same as a previous comment. Is there a reason we wouldn't want to group these?
```suggestion
        trackInteraction("toggle_detail", {
          direction: isDetailShown ? "collapse" : "expand",
        });
```